apps/frontend-manage/src/components/questions/QuestionPreviewModal.tsx

```diff
@@ -1,107 +0,0 @@
-import { useQuery } from '@apollo/client'
```

Comment:
Are we sure that we want to delete the preview modal completely? This would allow the lecturers to test the exact behavior of the student component incl. disabled buttons etc. with specified restrictions